modules/lambdas_secondary/main.tf

```diff
@@ -52,4 +52,30 @@ resource "aws_lambda_permission" "post_to_queue_and_trigger_step_function_allow_
   ## OPTIONAL
   statement_id = "AllowExecutionFromSQS"
   source_arn   = var.orca_sqs_s3_inventory_queue_arn
+}
+
+## Local Variables
+locals {
+  orca_bucket_names  = [for k, v in var.buckets : v.name if v.type == "orca"]
+}
+
+resource "aws_s3_bucket_inventory" "inventory-report" {
+  for_each = toset(local.orca_bucket_names)
+
+  bucket = each.key
+  name   = "${var.prefix}-${each.key}-inventory-report"
+
+  included_object_versions = "All"
+  optional_fields = ["Size", "LastModifiedDate", "StorageClass", "ETag"]
+
+  schedule {
+    frequency = var.s3_report_frequency
+  }
+
+  destination {
+    bucket {
+      format     = "CSV"
+      bucket_arn = var.orca_reports_bucket_arn
+    }
+  }
 }
```

Comment:
shouldn't this be under modules/s3/main.tf as this is relevant to s3?

Also I see modules/s3 is in the previous https://github.com/nasa/cumulus-orca/pull/259 but not in this one.

Reply:
Yup. Can't automate s3 bucket creation, so we're just doing this here.